build.sh

```diff
@@ -17,8 +17,9 @@ usage()
     echo
 }
 
-if [ $1 == "-?" ]; then
+if [ "$1" == "-?" ]; then
     usage
+    exit 0
```

Comment:
I'll prefer if you delete the exit code here. Just for consistency.
I'll work in the `build-managed.sh` usage separately